### PR TITLE
SDK-18

### PR DESCRIPTION
**Pull request by sameerkedia**

### CODE REVIEW

File: src/main/java/com/qubole/qds/sdk/java/details/ClusterApiImpl.java [hunk 1]
```diff
@@ -74,6 +75,13 @@
         RequestDetails entity = new RequestDetails(configBuilder.toString(), RequestDetails.Method.POST);
         return new GenericInvokableBuilderImpl<ClusterItem>(client, entity, ClusterItem.class, "clusters");
     }
+    
+    @Override
+    public InvokableBuilder<ClusterItem> delete(String labelOrId)
+    {
+        RequestDetails entity = new RequestDetails(null,RequestDetails.Method.DELETE);
```
Comment (mindprince) [inline src/main/java/com/qubole/qds/sdk/java/details/ClusterApiImpl.java]: Nit: Space after `null,`.

File: src/main/java/com/qubole/qds/sdk/java/details/ClusterApiImpl.java [hunk 2]
```diff
@@ -17,6 +17,7 @@
 
 import com.qubole.qds.sdk.java.api.*;
 import com.qubole.qds.sdk.java.client.QdsClient;
+import com.qubole.qds.sdk.java.details.RequestDetails.Method;
```
Comment (mindprince) [inline src/main/java/com/qubole/qds/sdk/java/details/ClusterApiImpl.java]: Why do we need this now?

**Comment by mindprince** [comment]
Merge the three examples into one.
The flow should be like:
1. Create cluster.
2. Show the created cluster.
3. Edit the created cluster.
4. Show the edited cluster.
5. Delete the cluster.

File: src/main/java/com/qubole/qds/sdk/java/details/ClusterApiImpl.java (hunk 1 above)
Comment (sameerkedia) [inline src/main/java/com/qubole/qds/sdk/java/details/ClusterApiImpl.java]: Okay, done.

File: src/main/java/com/qubole/qds/sdk/java/details/ClusterApiImpl.java (hunk 2 above)
Comment (sameerkedia) [inline src/main/java/com/qubole/qds/sdk/java/details/ClusterApiImpl.java]: Sorry, forgot to remove it. Removed.

**Comment by mindprince** [comment]
Squash merged this after making some edits: https://github.com/qubole/qds-sdk-java/commit/335fbfb691b689b106613574b64f0a9e5e35917b

**Pull request closed without merging**